gov_lib.py

```diff
@@ -6,10 +6,10 @@
 logger = logging.getLogger(__name__)
 
 def set_detz_in(gov_robot, distance):
```

Comment:
I think we might need to rename these functions, or make new functions for set_detz_work and set_detz_safe. Having set_detz_in set detz's work seems like it will come back later.

Reply:
I would prefer we keep it for now, since we're just going into the cycle. however, we will need to keep an eye on it for AMX/FMX and make sure that the position names match between all beamlines.

if you like, you can put in a github issue for us to do for 2022 cycle 2, where we give it more logical names (based on function in this case) instead of just position (in/out). if you do, please make a comment and link to it (#<number>)